drivers/usb/device/usb_dc_rpi_pico.c

```diff
@@ -635,15 +635,27 @@ int usb_dc_ep_read_continue(uint8_t ep)
 		return -EINVAL;
 	}
 
-	bytes_received = state.setup_available ?
-			 sizeof(struct usb_setup_packet) :
-			 udc_rpi_get_ep_buffer_len(ep);
+	bool data_packet_after_setup_is_expected = false;
```

Comment:
Move the declaration above to the declarations above to pass compliance check. Maybe rename to `arm_out_endpoint` but there can be multiple disagreements about the names...

Reply:
Sounds good to me, will use that name, or similar.

Reply:
Done, using arm_out_endpoint